lib/instance_billing_flavor_check/utils.py

```diff
@@ -104,7 +104,9 @@ def get_rmt_ip_addr():
     # use present RMT server IP first if registered
     for etc_hosts_line in etc_hosts_lines:
         if 'susecloud.net' in etc_hosts_line:
-            return etc_hosts_line.split('\t')[0]
+            etc_hosts_addrs = etc_hosts_line.split('\t')[0]
+            return [etc_hosts_addrs.split(' ')[0]]
```

Comment:
There could be more than 1 line if IPv6 is enabled, you need a loop

Reply:
so IIUC
```
<ipv4> smt-csp.susecloud.net smt-csp 
<ipv6> smt-csp.susecloud.net smt-csp
```
is valid

Reply:
will adapt the code accordingly